include/SDL3/SDL_events.h

```diff
@@ -167,7 +167,10 @@ typedef enum
     /* 0x800, 0x801, and 0x802 were the Gesture events from SDL2. Do not reuse these values! sdl2-compat needs them! */
 
     /* Clipboard events */
+    // TODO: Is there any point in keeping the original event here?
+    //       Perhaps just update the existing one?
     SDL_EVENT_CLIPBOARD_UPDATE = 0x900, /**< The clipboard or primary selection changed */
+    SDL_EVENT_CLIPBOARD_DATA_UPDATE,    /**< The clipboard changed (any mime-type) */
```

Comment:
This shouldn't be here. Forgot to remove it.

The problem with the SDL_CLIPBOARD_DATA_EVENT is that I'd like it to contain mime-types. But the list of mime-types for one offering can be long and I don't like the idea of allocating a `char**` just to transfer the data.